test/banners/mobile_english/components/BannerVar.spec.ts

```diff
@@ -22,7 +22,7 @@ let pageScroller: PageScroller;
 const formModel = useFormModel();
 const translator = ( key: string ): string => key;
 
-describe( 'BannerVar.vue', () => {
```

Comment:
Looks like there was a few changes caused by a PhpStorm refactor. You'll need to revert these.

Reply:
Done.